docs/docs/setup-guides/01_deploy-to-hosted-flex.md

```diff
@@ -1,31 +1,66 @@
 ---
-sidebar_label: Setup a release pipeline
+sidebar_label: Deploy to hosted Flex
 sidebar_position: 2
-title: Setup a release pipeline
+title: Deploy to hosted Flex
 ---
 
+*WARNING* : These deploy steps will inject taskrouter configuration overwriting resources that may already be there.  This includes
+
+Task Router - Task Queues:
+ - Everyone
+ - Sales
+ - Support
+ - Internal Calls
+
+Task Router - Workflows
+ - Assign To Anyone
+ - Chat Transfer
+ - Callback
+ - Internal Call
+
+Task Router - Activities
+ - Offline
+ - Available
+ - Unavailable
+ - Break
+
+Task Router - Task Channels
+ - Voice
+ - Chat
+
+Studio Flows
+ - Messaging Flow
+ - Chat Flow
+ - Voice IVR
+
+
 _~5 minutes_
 
-1. Use the template to create your own repository
+1. Use the template to [create your own repository](https://github.com/twilio-professional-services/flex-project-template/generate) 
+  - (Optionally) after creating your repo you may also want to attach the history to your new repository for future updates - details [here](/setup-guides/managing-future-updates-from-the-template)
```

Comment:
This needs one additional space before the hyphen in order to be spaced correctly.